### PR TITLE
Add a C API for the SEV launch ioctls

### PR DESCRIPTION
**Pull request by tylerfanelli**
Add a C FFI layer for the SEV launch ioctls. This API is hidden behind the `capi` feature, which is enabled when any `cargo-c` commands (`cbuild`, `ctest`, `cinstall`, etc) are enabled.

### CODE REVIEW

**Comment by tylerfanelli** [comment]
@larrydewey @DGonzalezVillal 

When you have the chance, could you review?

**Review by larrydewey — DISMISSED** [review]
LGTM just some discussion points.

**Review by DGonzalezVillal — APPROVED** [review]
Reviewed alongside @larrydewey

File: cbindgen.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+include_guard = "LIBSEV_H"
```
[the remark this answers is not in the document]
Comment (tylerfanelli) [reply cbindgen.toml]: Changed to `RUST_SEV_FFI_H`.

File: src/error.rs [hunk 2]
```diff
@@ -1,12 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use std::{
-    convert::From,
+    convert::{From, Into},
```
[the remark this answers is not in the document]
Comment (tylerfanelli) [reply src/error.rs]: Removed.

**Review by larrydewey — APPROVED** [review]

**Pull request merged**